### PR TITLE
Lucas change

### PR DESCRIPTION
**Pull request by carmaHowlett**
Please review this pull request to be merged if acceptable, @casey-collab

### CODE REVIEW

File: english/dog/meetdug.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"My name is Dug, I have just met you, and I love you!" the dog began to speak using
+a special collar made by his owner. The dog chased a squirrel so I turned
+back to the man and woman.
+
+[Talk to the woman](../woman/scully.md)
```
Comment (casey-collab) [inline english/dog/meetdug.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/dog/meetdug.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+[Talk to the man](../man/mulder.md)
```
Comment (casey-collab) [inline english/dog/meetdug.md]: This link is also invalid.

File: english/talk_to_neighbour/talk_to_neighbour.md [hunk 3]
```diff
@@ -7,3 +7,5 @@ As you speak to your neighbor, you see a dark car approach. Two figures emerge,
 [Talk to the woman](woman/scully.md)
 
 [Talk to the man](man/mulder.md)
+
+[Talk to the dog](dog/meetdug.md)
```
Comment (casey-collab) [inline english/talk_to_neighbour/talk_to_neighbour.md]: This link is also invalid.

File: english/dog/meetdug.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+"My name is Dug, I have just met you, and I love you!" the dog began to speak using
```
Comment (casey-collab) [inline english/dog/meetdug.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
It looks like the files `english/talk_to_neighbour/.DS_Store` and `english/.DS_Store` are empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to each file, or delete the options that lead to them.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/dog/meetdug.md (hunk 1 above)
Comment (carmaHowlett) [inline english/dog/meetdug.md]: How do I make it work correctly?

File: english/dog/meetdug.md (hunk 4 above)
Comment (carmaHowlett) [inline english/dog/meetdug.md]: okay, how can I fix this @casey-collab ?

**Pull request closed without merging**